### DISCUSSION TITLE
GBA Roms

### DISCUSSION THREAD

**Discussion by HJMBgit84** [rafaelvcaetano/melonDS-android#58] [Q&A]
I wonder how can I make the GBA games run in the emulator because I see the configuration to select the GBA ROM directory and the GBA save directory but I cannot select the directories, I only can select the direct ROM, would you please explain it?

Also want to ask if are any plans for landscape mode in the emulator please.

**Comment by aimdloop** [comment]
> I wonder how can I make the GBA games run in the emulator because I see the configuration to select the GBA ROM directory and the GBA save directory but I cannot select the directories, I only can select the direct ROM, would you please explain it?

I don't think this emulator can run gba roms.But it can be used in many games like pokemon where u can migrate older gen pokemon to newer gen pokemon means pokemon emerald to pokemon soul siver.
![Screenshot_2021-02-16-11-25-20-51_e7877f9796c3bfb5a5339f6b3513bed1](https://user-images.githubusercontent.com/47573399/108022223-5cb21e80-704a-11eb-9acd-612f2583e9a5.jpg)
![Screenshot_2021-02-16-11-26-38-49_e7877f9796c3bfb5a5339f6b3513bed1](https://user-images.githubusercontent.com/47573399/108022240-63409600-704a-11eb-8a51-ac70e762aed2.jpg)

**Comment by rafaelvcaetano** [comment]
Yep. It's just like @shabbirmahmud said. melonDS cannot play GBA games. However, there are multiple games that have additional features when certain GBA games are inserted.

Landscape mode will come with customizable layouts, which is a feature in development.

**Reply by HJMBgit84** [reply]
Ok ok now I get it, ok thanks so much for your reply and anxious to see the customized layout for landscape, if possible please add feature to edit touchscreen because the L and   R buttons are a little higher :) take your time and thanks so much once again.

**Comment by Snowball750** [comment]
Hey @rafaelvcaetano I did the same thing as @shabbirmahmud  and when I enter the Pokémon Heartgold menu, it does not appear that I can migrate

Is this a problem or something in melonds or am I doing something wrong?
